### ISSUE TITLE
formatting for MSDOS

### ISSUE THREAD

**Issue by robbo007** [ergonomy-joe/u6-decompiled#1]
Hi,
Not sure if its just me but the code seems to have lost the formatting for MSDOS. Line feeds and carriage returns. Any chance this could be rectified?

**Comment by Fenyx4** [comment]
I created a PR to try to fix this issue

**Comment by ergonomy-joe** [comment]
@robbo007 I am not sure to understand the problem.
I didn't check all the project's files but they all seem to have the CR/LF at the end of each line. I can edit them under DosBox and Windows, the build works fine under DosBox, ...
Could you be more specific about why this is an issue ?
@Fenyx4 seems to have added an extra CR for each line. Is this what you meant by "rectified" ?
Could you please clarify what you meant with "formatting for MSDOS"?

**Comment by robbo007** [comment]
@Fenyx4 Its the formatting which is all wrong under MSDOS. The files need to be edited using MSDOS edit and then saved. Then the format is corrected. Here is an example. A picture says more than a thousand words :)
https://ibb.co/tLVCd6c
https://ibb.co/9ckshw1

**Comment by robbo007** [comment]
imgbb is probably blocked from where you are? But at least you see the errors too. It's probably Github screwing with the files. Would be good to have them fixed. I don't want to convert them all in edit.com. LOL

**Comment by robbo007** [comment]
haha yes. Good eyes. :) This is a monumental piece of work by @ergonomy-joe. Can't wait for the rest of the Crown Jewels. :)

**Comment by ergonomy-joe** [comment]
@robbo007 @Fenyx4 Thanks you for the nice comments ^^;

Concerning the problem I think I finally figured out the origin: I'm using GitHub desktop to manage the repositories, and by default the option `autocrlf` is set to true in the application's `gitconfig`.
So when I first commited the files, they were all converted to "unix". And of course, the application ignored the line ending differences between my local files and those on github.
Sorry @Fenyx4, I won't be using your PR, I prefer to do that locally.
Thanks to you too for pointing out this issue
